Content.Server/StationEvents/Events/VentClogRule.cs

```diff
@@ -47,20 +46,34 @@ protected override void Started(EntityUid uid, VentClogRuleComponent component,
             if (!RobustRandom.Prob(Math.Min(0.33f * mod, 1.0f)))
                 continue;
 
+            string reagent;
             if (RobustRandom.Prob(Math.Min(0.05f * mod, 1.0f)))
             {
-                solution.AddReagent(RobustRandom.Pick(allReagents), 200);
+                reagent = RobustRandom.Pick(allReagents);
             }
             else
             {
-                solution.AddReagent(RobustRandom.Pick(component.SafeishVentChemicals), 200);
+                reagent = RobustRandom.Pick(component.SafeishVentChemicals);
             }
```

Comment:
I think this would look clearer as a single assignment with ternary condition for the reagent list. So something like this:

```c#
var safeChemical = RobustRandom.Prob(Math.Min(0.05f * mod, 1.0f));
var reagent = RobustRandom.Pick(safeChemical ? component.SafeishVentChemicals : allReagents);
```